include/flatbuffers/flatbuffers.h

```diff
@@ -1470,6 +1502,21 @@ class FlatBufferBuilder
     finished = true;
   }
 
+  // Allocates space for a vector of structures.
+  // Must be completed with EndVectorOfStructs().
+  template<typename T>
+  const T* StartVectorOfStructs(size_t vector_size) {
```

Comment:
Sure done, though I noticed a load of other stuff under protected that probably shouldn't be there.  Probably worth a future clean up.